### PR TITLE
Add network policy

### PR DESCRIPTION
**Pull request by corest**
Towards https://github.com/giantswarm/giantswarm/issues/4033

### CODE REVIEW

File: helm/net-exporter-chart/values.yaml [hunk 1]
```diff
@@ -4,6 +4,9 @@ namespace: monitoring
 userID: 1000
 groupID: 1000
 
+port: 8000
+giantswarmcidr: 137.117.188.104/32
```
Comment (rossf7) [inline helm/net-exporter-chart/values.yaml]:
```suggestion
giantswarmCIDR: 137.117.188.104/32
```

What is this network range? AIUI we can have comments in the values file and I think that is needed here. I don't follow what this is for.

Comment (corest) [reply helm/net-exporter-chart/values.yaml]: Added comment regarding that value

**Comment by corest** [comment]
PTAL

File: helm/net-exporter-chart/templates/np.yaml [hunk 2]
```diff
@@ -9,21 +9,45 @@ metadata:
 spec:
   podSelector:
     matchLabels:
-      app: {{ .Values.name }}
+      k8s-app: {{ .Values.name }}
```
Comment (calvix) [inline helm/net-exporter-chart/templates/np.yaml]: its not really k8s-app as it does not do anyhting with k8s functionality, (like kube-proxy or calico-node)

Comment (corest) [reply helm/net-exporter-chart/templates/np.yaml]: I think you’re commenting on some outdated config

**Comment by corest** [comment]
Now blocked by old kubernetes, used with minikube 0.28.0 in e2e-harness. That's k8s 1.10.0. 
@rossf7 what is our strategy on replacing e2e-harness with kind or whatever else?

**Comment by corest** [comment]
I've tested my changes. Fixing tests requires too much time. Merging

**Comment by rossf7** [comment]
> Fixing tests requires too much time.

@corest Ack this makes me sad too but I think it's the right call.

@puja108 @sslavic This is why I want us to work on the `basicapp` test and move to `kind`.Makes it much easier to keep up to date and we get logs for all pods, not just operator pods.

**Pull request merged**